examples/ansible.cfg

```diff
@@ -492,3 +492,30 @@
 
 # Set how many context lines to show in diff
 #context = 3
+
+[galaxy]
+# Controls whether the display wheel is shown or not
+#display_progress=
+
+# Validate TLS certificates for Galaxy server
+#ignore_certs = False
+
+# Role or collection skeleton directory to use as a template for
+# the init action in ansible-galaxy command
+#role_skeleton=
+
+# Patterns of files to ignore inside a Galaxy role or collection
+# skeleton directory
+#role_skeleton_ignore="^.git$", "^.*/.git_keep$"
+
+# Galaxy Server URL
+#server=https://galaxy.ansible.com
+
+# A list of Galaxy servers to use when installing a collection.
+#server_list=
```

Comment:
It might be a good idea to put an automation hub and galaxy server list here with the underlying sections for them as documented at https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client. I would probably only have `automation_hub` and `galaxy` as an option though and ignore the rest of those examples.